### PR TITLE
Fix spinner for student number and set min and max

### LINKED ISSUE #86: Disable the spinner for the student number input as well as set min and max value
http://stackoverflow.com/a/4298216/2094953

### PR DESCRIPTION
**Pull request by TimvdLippe**
Fixes #86

### CODE REVIEW

File: app/elements/login-popup.html [hunk 1]
```diff
@@ -90,7 +97,7 @@
                   label="Username" type="text" value="{{username}}"
                   on-keydown="enterAndTryRegister"></paper-input>
               <paper-input required auto-validate error-message="needs a number!"
-                  label="Student number" type="number" value="{{studentnumber}}"
+                  label="Student number" type="number" value="{{studentnumber}}" min=1000000 max=9999999
```
Comment (GijsWeterings) [inline app/elements/login-popup.html]: Maybe this is wishful thinking, but should we set the max as a variable to keep it easily changable?

File: bower.json [hunk 2]
```diff
@@ -15,12 +15,16 @@
     "akc-meta": "git://github.com/akc42/akc-meta.git#b0a6bde0e",
     "file-upload": "~1.6.1",
     "material-progress": "^0.1.2",
-    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2"
+    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2",
+    "paper-input": "timvdlippe/paper-input#spinner-mixin"
```
Comment (GijsWeterings) [inline bower.json]: we should probably make an issue to put this back to the upstream version after merging

File: app/elements/login-popup.html (hunk 1 above)
Comment (TimvdLippe) [reply app/elements/login-popup.html]: Doubt that is really necessary. TU won't run out of numbers any time soon.

File: bower.json (hunk 2 above)
Comment (TimvdLippe) [reply bower.json]: Done https://github.com/Mentoraat/plannings-tool-client/issues/89

File: app/elements/login-popup.html (hunk 1 above)
Comment (GijsWeterings) [reply app/elements/login-popup.html]: nah, but study numbers are at the rise fast, and may hit the max value at some point in a few years... but like I said, wishful thinking

**Comment by GijsWeterings** [comment]
had to move your changes around to the new file bc of wrong pr order (yay git skills... not). @TimvdLippe this is correct right?

File: app/elements/account-elements/register-page.html [hunk 3]
```diff
@@ -3,14 +3,22 @@
 <link rel="import" href="account-page.html">
 
 <dom-module id="register-page">
+  <style>
```
Comment (TimvdLippe) [inline app/elements/account-elements/register-page.html]: Should be inside the template

**Pull request merged**